osu.Game/Screens/Edit/Setup/ResourcesSection.cs

```diff
@@ -100,8 +100,14 @@ public bool ChangeAudioTrack(FileInfo source, bool applyToAllDifficulties)
 
             TagLib.File? tagSource;
 
+            // Thai language users had trouble uploading songs because the automatic parsing of TagLib#
+            // had problems because of the 'th' value of CurrentCulture. Switch to InvariantCulture while parsing the data.
+            // Alternatively: Provide MIME type of data in TagLib.File.Create()
```

Comment:
I would very much prefer any alternative to this. The osu! framework already manages thread culture and I very much do not want to see any game code other than that specific piece of framework touching culture. 

Things I would consider acceptable alternatives:

- Fixing the bug upstream (probably not viable because taglib doesn't look very maintained right now)
- Copying the specific bit responsible for picking the MIME from the extension and fixing the culture variance of that
- Just hardcoding mp3 format or something